### PR TITLE
Member check in

### LINKED ISSUE #294: Member check in

### PR DESCRIPTION
**Pull request by sambehrens**
closes #294

### CODE REVIEW

File: .prettierrc.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-tabWidth: 4
+tabWidth: 10
```
Comment (karlmolina) [inline .prettierrc.yml]: what the heck is this

**Review by karlmolina — DISMISSED** [review]

File: .prettierrc.yml (hunk 1 above)
Comment (sambehrens) [reply .prettierrc.yml]: i thought we should have wider tabs⃩

Comment (karlmolina) [reply .prettierrc.yml]: ☯🐚  Ｗ𝕚𝒹€ᖇ ⓉＡ𝐛s  😈🐍

**Comment by sambehrens** [comment]
the basic functionality for checking members in is here, we just need to make it so if you want to return to any other pages, it forces you to re-authenticate. I don't really know how we are going to do that right now an dit could take a long time to figure out, so I am going to keep this ticket manageable and just merge it in as it is

**Review by karlmolina — APPROVED** [review]

**Pull request merged**